modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,704 @@
+
+The new `addView` API registers a new view container.
+
+This container, which is off (hidden) by default, is attached to the main view.
+
+In this default state, the main editor view is shown and the `addView`-specific view is hidden.
+
+When `addView` is turned on, the main editor view is hidden, and the `addView`-specific view is shown.
+
+Accompanying this new API is a new `ToggleView` command.
+
+The `ToggleView` command can, as its name suggest, toggle view visibility.
+
+As well, the `ToggleView` command can be queried for its current state.
+
+
+=== New color_default_foreground and color_default_background options to set the initial default color for the forecolor and backcolor toolbar buttons and menu items.
+
+{productname} 6.3 adds a new feature that allows the default color for the `color_default_foreground` and `color_default_background` in the toolbar button menu to be pre-defined.
+
+The `color_default_foreground` and `color_default_background` colors are pre-defined by including them in `tinymce.init`
+
+In the example below, the foreground color is set to red and the background color is set to a shade of cyan.
+
+As the example shows, these colors can be defined by either name or hex #.
+
+[source,js]
+----
+<script type="text/javascript">
+tinymce.init({
+    selector: "textarea",
+	color_default_background: 'red',
+	color_default_foreground: '#0DEDFF',
+	toolbar: 'forecolor backcolor fontsize'
+});
+</script>
+----
+
+=== Placing block elements inside anchor tags is now supported.
+
+Previously, {productname} only supported placing inline elements — such as `+<strong></strong>+` or `+<code></code>+` — inside anchor tags
+
+{productname} 6.3 adds support for wrapping anchor tags around block level elements.
+
+For example, the following is now supported:
+
+[source,html]
+----
+<a href="https://example.com/"><p>Link text inside <code>&lt;p&gt;</code> tags.</p></a>
+----
+
+=== Added `ToggleToolbarDrawer` event to subscribe to toolbar’s opening and closing. 
+
+{productname} 6.3 includes a new event, `ToggleToolbarDrawer`, to subscribe to the toolbar's opening and closing actions.
+
+In previous versions of {productname}, when the `ToggleToolbarDrawer` command was set to `toolbar_mode: "sliding"` or `toolbar_mode: “floating”,` the `ToggleToolbarDrawer` did not create an opening or closing event.
+
+With {productname} 6.3, the new `ToggleToolbarDrawer` event has been introduced. When the `ToggleToolbarDrawer` command is triggered, the new event initializes an `onToggle` callback that propagates the `OuterContainer` component for `SplitSlidingToolbar` and `SplitFloatingToolbar`.
+
+Once the `OuterContainer` component event is triggered, the console records either a `true` or `false` action that calls on the `onToggled` callback which supports the `FloatingToolbarButton` and `SplitFloatingToolbar` components.
+
+
+[[changes]]
+== Changes
+
+{productname} 6.3 also incorporates the following changes:
```

Comment:
It still seems odd that we are separating this from the other color picker things because this is dependent on that first change, but I'll just accept it and move on.